packages/profiling-browser/src/utils.ts

```diff
@@ -0,0 +1,400 @@
+import { WINDOW } from '@sentry/browser';
+import type {
```

Comment:
All of this is mostly about constructing the profile envelope and converting from JS sampled format to the one we expect in Sentry